assets/js/blocks/checkout/block.tsx

```diff
@@ -84,22 +84,29 @@ const Checkout = ( {
 		return <CheckoutOrderError />;
 	}
 
+	/**
+	 * Show the login prompt if the user is not logged in
```

Comment:
I like the comment here, useful to see what's going on below

Reply:
Thanks, @alexflorisca. I've added the comment, which @mikejolley improved, as the check below is a bit confusing without context.